src/containers/Masthead/__tests__/MastheadSearchForm-test.js

```diff
@@ -19,7 +19,7 @@ test('MastheadSearchForm redirects on ndla url paste with id at the end', () =>
     push: sinon.spy(),
```

Comment:
Ikke at det er lagt til nå, men muligheter for å bytte ut til jest.spyOn isteden hvis mulig? 

https://jestjs.io/docs/en/jest-object#jestspyonobject-methodname